netclient/cli_options/flags.go

```diff
@@ -204,5 +204,23 @@ func GetFlags(hostname string) []cli.Flag {
 			Value:   "no",
 			Usage:   "Allows to run the command with force, if otherwise prevented.",
 		},
+		&cli.BoolFlag{
+			Name:    "verbosity-level-1",
+			Aliases: []string{"V"},
```

Comment:
upper case?  isn't lower case more common.

Reply:
yeah you're right, I'll change to lowercase, I was running into issues with lowercase at some point and left at uppercase for some reason